magento/merchdocs#1122: Describe steps to allow remote shopping assistance (#1123)
magento/merchdocs: src/customers/login-as-customer.md

Enabling Login as Customer requires that you enable the feature in your Magento instance and then enable access for Magento Admin users in the user role permissions.

> ## Customer permission for remote shopping assistance

Reviewer: ```suggestion
## Customer account permission for remote shopping assistance
```

> The Magento Admin can login as a customer, only after the customer allows remote access.

Reviewer 2: Thank you for your contributions, @ZhovnirMaria 

I would like to simplify a bit your sentence and add a preliminary explanations to steps below. Something like that:

```
To allow the ability to login as a customer for Magento admin user, a customer should make the next steps:
```

Thanks

Author: Hi, @atwixfirster
I made some changes to the request, you can check them.
Thank you!

Reviewer 2: Thank you, @ZhovnirMaria !

@dobooth , could you please review if the suggested changes are correct here?

Thank you!

CC: @leslietilling

> To allow the ability to login as a customer for Magento admin user, a customer should make the next steps:

Reviewer: ```suggestion
To allow the ability for the merchant to log in as the customer from the Admin, a customer should take these steps:
```

1. Go to the **Account Information** page.

> 1. Check the **Allow remote shopping assistance** checkbox.

Reviewer: ```suggestion
1. Select the **Allow remote shopping assistance** checkbox.
```

1. Click the <span class="btn">Save</span> button.

![]({% link images/images/permission.png %}){: .zoom}
_Account Information Page_

> Without this permission, Magento Admin cannot login as this customer.

Reviewer: ```suggestion
Without this permission, an Admin user cannot login as this customer to provide assistance.
```
